### PR TITLE
Rate table & snowglobes interface update

### PR DESCRIPTION
**Pull request by evamick**
To the rate table code, added KM3NeT, integrated DarkSide-20k, XENONnT, LZ, and PandaX-4T to be calculated like the rest of the detectors, and fill in spots that were previously "-". As well as formatting it to output formatted to create an html table utilizing mathjax. Also had to add LZ & PandaX-4T into the snowglobes_interface.py for those integrations to work. I added the html rate table code as its own file rather than replacing the existing one, but that could be changed if desired!

### CODE REVIEW

File: snowglobes_interface.py
Comment (evamick) [inline snowglobes_interface.py:1]: I initially uploaded an updated copy of this file rather than just editing the file to have the updates

**Comment by evamick** [comment]
I haven't done a pull request before, I'm sorry if anything isn't done correctly!

File: python/snewpy/scripts/SNEWS2.0_rate_table.py [hunk 1]
```diff
@@ -1,190 +1,193 @@
...
-
-
-ascii.write(data,Writer=ascii.Latex)
+# +50 in line 143 is for rounding so we have a nice number for the table
+
+dettype='km3net'
+mass=69366 * 3
+data['Mass [kt]'][3] = "~"+str(100*int(0.01*(mass*total_events['s27.0']['AdiabaticMSW_NMO'][dettype+"smeared"]/
+      total_events['s27.0']['AdiabaticMSW_NMO'][dettype+"unsmeared"]))+50)+"*"
+# +50 in line 145 to get 150 as an output, the average between normal (100) & inverted (200)
```
Comment (JostMigenda) [inline python/snewpy/scripts/SNEWS2.0_rate_table.py:150]: Isn’t this the same rounding as for `icecube` a few lines above?

File: python/snewpy/scripts/SNEWS2.0_rate_table.py [hunk 2]
```diff
@@ -1,190 +1,193 @@
...
-data['Type'].append('Xe/any $\\nu$')
-data['Mass [kt]'].append(mass)
-data['Location'].append('China')
-data['\\SI{11.2}{\solarmass}'].append(int(mass*s11_epT))
-data['\\SI{27.0}{\solarmass}'].append(int(mass*s27_epT))
-data['\\SI{40.0}{\solarmass}'].append("-")
-
-
-ascii.write(data,Writer=ascii.Latex)
+# +50 in line 143 is for rounding so we have a nice number for the table
```
Comment (JostMigenda) [inline python/snewpy/scripts/SNEWS2.0_rate_table.py:144]: There’s an old adage: “Don’t comment bad code—rewrite it!”

Your comment definitely helps; but I think it’d be even better to change this to use the built-in `round()` function, and get rid of the puzzling `int(0.01*(… + 50))`.

Comment (JostMigenda) [inline python/snewpy/scripts/SNEWS2.0_rate_table.py:144]: There’s an old adage: “Don’t comment bad code—rewrite it!”

Your comment definitely helps; but I think it’d be even better to change this to use the built-in round() function, and get rid of the puzzling `int(0.01*(… + 50))`.

File: python/snewpy/scripts/SNEWS2.0_rate_table.py (hunk 1 above)
Comment (mcolomermolla) [reply python/snewpy/scripts/SNEWS2.0_rate_table.py:150]: We are not sure why it was there for IceCube, and for IceCube, it is there only inside the denominator (unsmeared), not out of the parenthesis...

**Comment by JostMigenda** [comment]
This is looking quite good now; thanks for your work!

Just one more thing: Since Mathjax and LaTeX are so similar, would it be possible to extend the code to output either format (as selected by the user)? LaTeX tables are handy for copying and pasting into papers, so it’d be great to keep that functionality without extra work by the user.
(I think apart from the HTML tags before/after the Mathjax, the changes would be very minor.)

**Review by JostMigenda — APPROVED** [review]
Thanks @evamick for the good work and congratulations on your first contribution to snewpy! 🎉

**Pull request merged**